pyproject.toml

```diff
@@ -44,6 +44,7 @@ classifiers = [
 cli = ["click>=8.1.7", "pyogrio>=0.8", "shapely>=2"]
 geopandas = ["geopandas>=0.13", "pandas>=2", "shapely>=2"]
 movingpandas = ["movingpandas>=0.17"]
+explore = ["mapclassify>=2.8.1"]
```

Comment:
I think we can include this in the `geopandas` extra

Reply:
ok, wasnt sure how you wanted to handle that part. Geopandas itself only has a soft dependency on mapclassify so its only imported when `scheme` is used; wasn't sure if you wanted to keep it similarly isolated (even though I hadn't structured the import like that)

Reply:
Hmm, in that case, maybe it is better to keep it separate. Especially if we move the `mapclassify` import only within the `else` clause where it's used. 

In that case, it's probably fine to not even define `mapclassify` in an extra.

Reply:
I removed this extra in favor of producing a runtime error